oktapam/resource_project_group.go

```diff
@@ -237,14 +235,30 @@ func resourceProjectGroupDelete(ctx context.Context, d *schema.ResourceData, m i
 	return nil
 }
 
-func createProjectGroupResourceID(project string, group string) string {
-	return fmt.Sprintf("%s|%s", project, group)
-}
-
 func parseProjectGroupResourceID(resourceId string) (string, string, error) {
```

Comment:
Should this method still be used? On [L159](https://github.com/okta/terraform-provider-oktapam/pull/69/files#diff-77c05ff4b5908fd5ef630fcab05f0286bb1655543d1c266b4ccce8ecf099278fR159) the ID is being set to the UUID.

Reply:
Importer handler function calls parse function. Concatenated attributes format is still required for importing because project group cannot be fetched with UUID.